ci-operator/templates/openshift/installer/cluster-launch-installer-openstack-e2e-ppc64le.yaml

```diff
@@ -471,7 +471,6 @@ objects:
 
         OS_DEBUG=1 TF_LOG=trace openshift-install --dir=/tmp/artifacts/installer create cluster --log-level=debug &
         PID_OPENSHIFT_INSTALL=$!
-        echo "PID_OPENSHIFT_INSTALL=${PID_OPENSHIFT_INSTALL}"
```

Comment:
Why drop this from up here?

Reply:
For consistency around the wait.  And so that it doesn't get lost from the copy files code block.